oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java

```diff
@@ -173,6 +173,9 @@ public APIGatewayProxyResponseEvent authCodeRequestHandler(
             return generateApiGatewayProxyErrorResponse(e.getStatusCode(), e.getErrorResponse());
         }
 
+        LOG.info("orchSession: {}", orchSession);
```

Comment:
Looks like your temp logging for testing is still here

Reply:
I can't test in dev so I was going to pause promotions from staging to test in staging, then remove the logging

Reply:
Ah gotcha. I'm inclined to include TODO comments to make it clear (mainly to myself) they're to be removed soon, but that's just what's helpful to me, and these will be removed immediately, so not a problem for anyone else seeing these 👍